pkg/storage/gcp/lister.go

```diff
@@ -17,7 +18,8 @@ func (s *Storage) List(ctx context.Context, module string) ([]string, error) {
 	ctx, span := observ.StartSpan(ctx, op.String())
 	defer span.End()
 
-	it := s.bucket.Objects(ctx, &storage.Query{Prefix: module})
+	modulePrefix := filepath.Join(module, "/@v")
```

Comment:
Isn't filepath.Join Os dependent? On Windows it will use `\` instead of `/`.

Reply:
@marpio great catch, I live in my own world of `/` so it didn't occur to me. Honestly, it's a good point for windows testing in CI :) -- anywho I updated the code. Let me know if there's anything else, thanks!